runtime/vm/jvminit.c

```diff
@@ -1890,7 +1890,7 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			argIndex = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXUSECONTAINERSUPPORT, NULL);
 			argIndex2 = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXNOUSECONTAINERSUPPORT, NULL);
 
-			if (argIndex > argIndex2) {
+			if ((-1 == argIndex2) || (argIndex > argIndex2)) {
```

Comment:
If neither option is specified, the `argIndex`'s will be the same.  Isn't it sufficient to do `if (argIndex >= argIndex2)` plus add a comment to indicate that `-XX:+ContainerSupport` is the default?

Reply:
that would work as well.